.env

```diff
@@ -0,0 +1 @@
+DB_PASSWORD=3Ik9fSJZ9o7j9DYJ
```

Comment:
oh no.

Reply:
Good catch, don't know why this happened, it has been removed